ephios/settings.py

```diff
@@ -335,3 +335,14 @@ def GET_SITE_URL():
     },
     "REFRESH_TOKEN_EXPIRE_SECONDS": 60 * 60 * 24 * 31 * 3,  # 3 months
 }
+
+if ENABLE_OIDC_CLIENT := env.bool("ENABLE_OIDC_CLIENT", False):
+    INSTALLED_APPS.append("mozilla_django_oidc")
+    AUTHENTICATION_BACKENDS.append("mozilla_django_oidc.auth.OIDCAuthenticationBackend")
+    OIDC_RP_CLIENT_ID = env.str("OIDC_RP_CLIENT_ID")
+    OIDC_RP_CLIENT_SECRET = env.str("OIDC_RP_CLIENT_SECRET")
+    OIDC_RP_SIGN_ALGO = env.str("OIDC_RP_SIGN_ALGO")
+    OIDC_OP_AUTHORIZATION_ENDPOINT = env.str("OIDC_OP_AUTHORIZATION_ENDPOINT")
+    OIDC_OP_TOKEN_ENDPOINT = env.str("OIDC_OP_TOKEN_ENDPOINT")
+    OIDC_OP_USER_ENDPOINT = env.str("OIDC_OP_USER_ENDPOINT")
+    OIDC_OP_JWKS_ENDPOINT = env.str("OIDC_OP_JWKS_ENDPOINT")
```

Comment:
that looks complicated to configure. I think we should create a docs page linking to mozillas OIDC documentation...

Reply:
the problem is that all of these values need to come from the OP, so the docs there also say little apart from "use the values that you get from your OP"

Reply:
Yeah I know. I just think we should create a docs page for these kind of features.

Reply:
done